### DISCUSSION TITLE
It's a pity the lack of documentation

### DISCUSSION THREAD

**Discussion by NormandoHall** [telegraf/telegraf#1512] [General]
This project is superb. Thanks to developers for this!

But the newcomers as me, spent lot of time trying to understand every aspect of this library. I think this can be more popular than now it is, if has better documentation.

Looking at https://telegraf.js.org it is pure API code, and not very helpful. I know this is a community effort and not only developers are responsible for this lack of documentation, but they knows better than anyone this code. Maybe if they start writing new documentation the community can follow and help to upgrade it.

Please, don't blame me, it is only an opinion.

Regards

**Comment by aradalvand** [comment]
9 months later and still no real improvement has been done to the documentation.
Such a bummer...
